app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainFragment.kt

```diff
@@ -9,24 +9,22 @@ import edu.uc.kovaciad.slicetracker.R
 import edu.uc.kovaciad.slicetracker.dto.Printer
 
 
-class MainFragment : Fragment() {
+class MainFragment : SuperFragment() {
```

Comment:
Changing subFragments to extend the newly created super fragment as shown in this class material video here: https://www.youtube.com/watch?v=pwZQRw08mP0